static/jade/getting-here/index.jade

```diff
@@ -157,23 +157,23 @@ block viewport
 
     p
       ul
-        li 350 CZK (~13 EUR) per night per each person in a double room
+        li 350 CZK (~13 EUR) per night per person in a double room
```

Comment:
I wanted to distinguish "per person" and "per 1 person", which is on the other line